dataprofiler/profilers/profile_builder.py

```diff
@@ -1312,28 +1312,33 @@ def report(self, report_options=None):
         omit_keys = report_options.get("omit_keys", [])
         num_quantile_groups = report_options.get("num_quantile_groups", 4)
 
-        columns = list(self._profile.values())
+        report_schema = dict()
+        for i in range(len(self._profile)):
+            col = self._profile[i].name
+            report_schema.setdefault(col, []).append(i)
+
```

Comment:
since we already loop through profile below, should we just calculate the `profile_schema` in that loop?

Reply:
Good idea, will add